qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -268,13 +290,16 @@ def _get_operated_qubits(self) -> Dict[int, List[int]]:
         operated_qubits = defaultdict(list)
 
         # Single qubits
-        for qubit in self._qubits:
-            operated_qubits[1].append([qubit])
+        if self._coupling_map:
+            for qubit in set(qubit for coupled in self._coupling_map for qubit in coupled):
+                operated_qubits[1].append([qubit])
```

Comment:
I reverted this here [aacba9a](https://github.com/Qiskit/qiskit-experiments/pull/740/commits/aacba9a665a7b8c07b451674732b15c54aab8586) since in `_get_full_qubits_of_schedule` we do `len(partial_qubits) == circuit_inst_num_qubits` i.e. we expect an object with a `len` and this change produces an `int`

Reply:
Oh, I missed the square brackets. I guess it would be

```suggestion
            operated_qubits[1] = [[q] for q in set(qubit for coupled in self._coupling_map for qubit in coupled)]
```

but that's less of an improvement since it still has a loop in the comprehension.